CONTRIBUTING.md

```diff
@@ -42,6 +42,32 @@ Watch mode:
 yarn test:watch # unit tests watch mode, no lint
 ```
 
+## Functional tests
+
+We have one functional test ensuring that when we instantiate a full app with
+a `searchBox`, we are able to use it and get different hits than initially.
+
+Functional tests are built with [webdriver.io](http://webdriver.io/), a wrapper
+around [WebDriver API](https://www.w3.org/TR/2013/WD-webdriver-20130117/) (Selenium).
+On travis they will run on different browsers using [Sauce Labs](https://saucelabs.com/).
+
+Locally it will use a docker image bundling Selenium and browsers.
+
+```sh
+docker pull elgalu/selenium
+# this helps in loading the host website from the container,
+# comes from https://docs.docker.com/docker-for-mac/networking/#known-limitations-use-cases-and-workarounds
+sudo ifconfig lo0 alias 10.200.10.1/24
+docker run -d --name=grid -p 4444:24444 -p 6080:26080 -p 5900:25900 \
+    -e TZ="US/Pacific" -e NOVNC=true -v /dev/shm:/dev/shm --privileged elgalu/selenium
+npm run test:functional
```

Comment:
Yes this is the basic setup locally to launch the functional tests. I will had more details.